lib/cli/src/commands/app/purge_cache.rs

```diff
@@ -0,0 +1,54 @@
+//! Get information about an edge app.
+
+use super::util::AppIdentOpts;
+use crate::{
+    commands::AsyncCliCommand,
+    opts::{ApiOpts, ItemFormatOpts},
+};
+
+/// Purge caches for applications.
+///
+/// Cache scopes that can be cleared:
+/// * InstaBoot startup snapshots
+///   Will delete all existing snapshots.
+///   New snapshots will be created automatically.
+#[derive(clap::Parser, Debug)]
+pub struct CmdAppPurgeCache {
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub api: ApiOpts,
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub fmt: ItemFormatOpts,
+
+    #[clap(flatten)]
+    #[allow(missing_docs)]
+    pub ident: AppIdentOpts,
```

Comment:
This should not be required, by default we should check if there's an `app.yaml` in the current dir and get the app related to it in case it exists, so the user doesn't need to worry about the app id.

Also this should be an optional, and be called id:
```
    pub id: Optional<AppIdentOpts>,
```

Reply:
If you read the code, you will realize that this already happens.

Reply:
> Also this should be an optional, and be called **id**

Way clearer for the user

Reply:
Renamed to `app`